README.md

```diff
@@ -68,6 +68,30 @@ def loadUser[F[_]]
 }
 ```
 
+## Modules
+
++ [freestyle](http://frees.io/docs/) - Core module including building blocks for boilerplate free FP programs and apps over Free monads and cats.
+
++ [tagless](http://frees.io/docs/core/tagless/) - An alternative encoding to Free based on Tagless Final.
+
++ [effects](http://frees.io/docs/effects/) - MTL style effects such as reader, writer, state, error, and more modeled as free algebras. 
+
++ [fetch](http://frees.io/docs/integrations/fetch/) - Integration with the Fetch library for effitient data access from heterogenous datasources.
+ 
++ [fs2](http://frees.io/docs/integrations/fs2/) - Integration to run fs2 Streams in Freestyle programs.
+
++ [monix](http://frees.io/docs/integrations/monix/) -  Instances and utilities to interpret to `monix.eval.Task`.
+
++ [logging](http://frees.io/docs/patterns/logging/) - A purely functional logging algebra over Verizon's Journal. 
+
++ [slick](http://frees.io/docs/integrations/slick/) - Embedding of DBIO actions in Freestyle programs.
+
++ [doobie](http://frees.io/docs/integrations/doobie/) - Embedding of Doobie ConnectionIO actions in Freestyle programs.
+
++ [cache](http://frees.io/docs/effects/Cache/) - A generic cache with an in memory and redis based implementations.
```

Comment:
Either "with an in memory and a redis based implementation" or "with in memory and redis based implementations"